tripy/tripy/frontend/trace/ops/resize.py

```diff
@@ -33,8 +34,7 @@ class Resize(BaseTraceOp):
     scales: Optional[Sequence[float]]
     align_corners: bool
 
-    def infer_rank(self):
-        self.outputs[0].rank = self.inputs[0].rank
+    infer_rank = op_utils.InferRankPolicies.same_as_input()
```

Comment:
Isn't output rank same as length of second input (shape tensor) for resize?

Reply:
Resize can't change the rank of the input (only its shape) and the second input may not always be materialized at this point (e.g. when using scales). So it's just easier to get the rank from the first input.